FinniversKit/Sources/Fullscreen/MotorTransactionView/MotorTransactionButton/MotorTransactionButton.swift

```diff
@@ -6,6 +6,7 @@ public enum MotorTransactionButton: String {
     case flat = "FLAT"
     case callToAction = "CALL_TO_ACTION"
     case republishAd = "REPUBLISH_AD"
+	case confirmInsurance = "CONFIRM_INSURANCE"
```

Comment:
Is it something weird about the indentation here, or is it just GitHub?

Reply:
I commited this before I changed from tabs to spaces in Xcode 😅